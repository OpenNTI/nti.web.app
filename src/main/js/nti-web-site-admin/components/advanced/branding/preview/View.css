.preview-dialog {
	width: 768px;
}

.preview-dialog ul {
	list-style: none;
	padding: 0;
	margin: 0;
}

.preview-dialog > ul > li {
	box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.15);
	margin-bottom: 2rem;
}

.title {
	color: white;
	font-size: 1.125rem;
	margin: 0.5em 0;
}

.preview-dialog :global(.icon-light-x) {
	background: white;
	border-radius: 50%;
	font-size: 1.75rem;
	padding: 0.75rem;
	position: absolute;
	right: calc(100% + 2rem);
	cursor: pointer;
}

.controls {
	text-align: center;
}
