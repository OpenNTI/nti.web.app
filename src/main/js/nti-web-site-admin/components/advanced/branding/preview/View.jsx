import React from 'react';
import PropTypes from 'prop-types';
import {Button, Prompt} from '@nti/web-commons';
import classnames from 'classnames/bind';

import t from '../sections/assets/strings';
import {types} from '../sections/assets/constants';

import getWidget from './widgets';
import styles from './View.css';

const cx = classnames.bind(styles);

export default function Preview ({onSave, onClose}) {
	return (
		<Prompt.Dialog>
			<div className={cx('preview-dialog')}>
				<span className="icon-light-x" onClick={onClose} />
				<ul>
					{
						Object.values(types).map(type => {
							const Widget = getWidget(type);
							return !Widget ? null : (
								<li key={type}>
									<div className={cx('title')}>{t(['types', type, 'title'])}</div>
									<Widget />
								</li>
							);
						})
					}
				</ul>
				{ onSave && (
					<div className={cx('controls')}>
						<Button onClick={onSave}>Apply Changes</Button>
					</div>
				)}
			</div>
		</Prompt.Dialog>
	);
}
