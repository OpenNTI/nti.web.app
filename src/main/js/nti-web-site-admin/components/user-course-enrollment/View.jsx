import {Router, Route} from '@nti/web-routing';// eslint-disable-line

import Overview from './overview';
import Reports from './reports';
import Progress from './progress';
import Frame from './Frame';

export default Router.for([
	Route({path: '/reports', component: Reports, name: 'site-admin.users.user-course-enrollment.reports'}),
	Route({path: '/progress', component: Progress, name: 'site-admin.users.user-course-enrollment.progress'}),
	Route({path: '/', component: Overview, name: 'site-admin.users.user-course-enrollment.overview'})
], {frame: Frame});
