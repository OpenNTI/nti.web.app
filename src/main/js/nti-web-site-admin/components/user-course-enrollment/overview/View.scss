@import '~@nti/style-common';

.site-admin-user-enrollment-overview {
	.date-info {
		display: flex;

		.joined {
			margin: 0 10px 10px 0;
			width: 370px;

			.site-admin-card {
				background-color: #3fb34f;
				color: white;
			}
		}

		.last-activity {
			width: 370px;
		}
	}

	.active-times-widget {
		margin: 10px 0 0;
	}

	.activity-and-progress {
		display: flex;
		flex-direction: row;
		align-items: flex-start;

		.user-course-enrollment-progress {
			margin: 10px 0 10px 10px;
			width: 240px;

			.circular-progress {
				width: 200px;
				margin: 0 auto;
			}
		}
	}
}
