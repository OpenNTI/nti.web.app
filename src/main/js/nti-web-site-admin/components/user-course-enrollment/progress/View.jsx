import React from 'react';
import PropTypes from 'prop-types';
import {Progress} from '@nti/web-course';

import Card from '../../common/Card';

SiteAdminUserEnrollmentProgress.propTypes = {
	course: PropTypes.object,
	enrollment: PropTypes.object
};
export default function SiteAdminUserEnrollmentProgress ({course, enrollment}) {
	return (
		<Card>
			{course && enrollment && (
				<Progress.Overview course={course} enrollment={enrollment} singleItem />
			)}
		</Card>
	);
}
