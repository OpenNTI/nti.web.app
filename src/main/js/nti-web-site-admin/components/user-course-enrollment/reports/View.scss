@import '~@nti/style-common';

.site-admin-user-overview {
	@include clearfix;

	& > * {
		float: left;
		padding: 0 0.5rem 0.5rem 0;
	}

	.joined {
		width: 50%;

		.site-admin-date-value {
			background-color: var(--primary-green);
			color: white;
		}
	}

	.last-login {
		width: 50%;
	}
}
