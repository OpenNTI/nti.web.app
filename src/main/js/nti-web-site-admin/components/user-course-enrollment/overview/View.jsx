import './View.scss';
import React from 'react';
import PropTypes from 'prop-types';
import {Loading} from '@nti/web-commons';
import { Widgets } from '@nti/web-reports';

import DateValue from '../../common/DateValue';

import ActiveTimes from './ActiveTimes';
import LastActivity from './LastActivity';
import Progress from './Progress';

const { ActiveDays } = Widgets;

export default class SiteAdminUserEnrollmentView extends React.Component {
	static propTypes = {
		enrollment: PropTypes.object
	}

	render () {
		const { enrollment } = this.props;

		if(!enrollment) {
			return <Loading.Mask/>;
		}

		return (
			<div className="site-admin-user-enrollment-overview">
				<div className="date-info">
					<div className="joined">
						<DateValue date={enrollment.getCreatedTime()} label="Enrolled" />
					</div>
					<div className="last-activity">
						<LastActivity enrollment={enrollment}/>
					</div>
				</div>
				<ActiveDays entity={enrollment}/>
				<div className="activity-and-progress">
					<ActiveTimes enrollment={enrollment}/>
					{enrollment.CourseProgress && (<Progress enrollment={enrollment} />)}
				</div>
			</div>
		);

	}

}
