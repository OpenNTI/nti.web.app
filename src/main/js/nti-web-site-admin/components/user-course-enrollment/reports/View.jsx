import './View.scss';
import React from 'react';
import PropTypes from 'prop-types';

import { List } from '@nti/web-reports';

import Card from '../../common/Card';

SiteAdminUserEnrollmentReports.propTypes = {
	enrollment: PropTypes.object,
};
export default function SiteAdminUserEnrollmentReports({ enrollment }) {
	return enrollment ? (
		<Card>
			<List context={enrollment} />
		</Card>
	) : null;
}
