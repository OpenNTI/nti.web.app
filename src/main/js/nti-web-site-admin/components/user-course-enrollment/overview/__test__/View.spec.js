import React from 'react';
import renderer from 'react-test-renderer';

import * as TestUtils from '@nti/web-client/test-utils';
import { Date as DateUtils } from '@nti/lib-commons';

import View from '../View';

const { tearDownTestClient, setupTestClient } = TestUtils;
const { MockDate } = DateUtils;

const getMockService = hasData => {
	return {
		get: () => {
			if (hasData) {
				return {
					WeekDays: {
						Friday: [
							0,
							0,
							4,
							7,
							4,
							2,
							8,
							5,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
						],
						Monday: [
							3,
							5,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							3,
							6,
							0,
							0,
							0,
							15,
							5,
							2,
							1,
							0,
							0,
						],
						Saturday: [
							4,
							6,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
						],
						Sunday: [
							10,
							4,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
						],
						Thursday: [
							9,
							2,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							4,
							5,
							0,
							2,
							1,
							0,
							0,
							6,
							0,
							0,
						],
						Tuesday: [
							8,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							1,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
						],
						Wednesday: [
							5,
							6,
							7,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							0,
							6,
							3,
							0,
							0,
							2,
							0,
							0,
							0,
							0,
							0,
						],
					},
				};
			}

			return {};
		},
		getBatch: () => {
			return {
				getLink: () => 'mockActiveTimes',
			};
		},
	};
};

const onBefore = hasData => {
	jest.useFakeTimers();
	setupTestClient(getMockService(hasData));
};

const onAfter = () => {
	tearDownTestClient();
};

const flushPromises = () => new Promise(resolve => setImmediate(resolve));

/* eslint-env jest */
describe('Site admin user course enrollment overview view', () => {
	const actingDate = new Date('10/31/2018');
	MockDate.install(actingDate);

	beforeEach(() => onBefore(true));
	afterEach(onAfter);

	afterAll(() => {
		MockDate.uninstall();
	});

	test('Basic render test', async () => {
		const enrollment = {
			getLink: () => 'mockAnalytics',
			getLastSeenTime: () => new Date('10/30/2017'),
			getCreatedTime: () => new Date('10/20/2017'),
			CourseProgress: {
				PercentageProgress: 0.55,
			},
		};

		const cmp = renderer.create(<View enrollment={enrollment} />);

		jest.runAllTimers();
		await flushPromises();
		jest.runAllTimers();

		const tree = cmp.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
