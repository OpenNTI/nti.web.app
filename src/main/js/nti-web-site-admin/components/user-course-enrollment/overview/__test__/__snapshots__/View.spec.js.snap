// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site admin user course enrollment overview view Basic render test 1`] = `
<div
  className="site-admin-user-enrollment-overview"
>
  <div
    className="date-info"
  >
    <div
      className="joined"
    >
      <div
        className="site-admin-card site-admin-labeled-value site-admin-date-value"
      >
        <div
          className="label"
        >
          Enrolled
        </div>
        <div
          className="value"
        >
          <time
            className="nti-text date"
            dateTime="2017-10-20T00:00:00-05:00"
          >
            October 20, 2017
          </time>
        </div>
      </div>
    </div>
    <div
      className="last-activity"
    >
      <div
        className="site-admin-card site-admin-labeled-value site-admin-date-value"
      >
        <div
          className="label"
        >
          Last Active
        </div>
        <div
          className="value"
        >
          <time
            className="nti-text date"
            dateTime="2017-10-30T00:00:00-05:00"
          >
            Oct 30, 2017 12:00 AM
          </time>
        </div>
      </div>
    </div>
  </div>
  <div
    className="admin-active-days"
  >
    <div
      className="daily-activity-header"
    >
      Daily Activity
    </div>
    <div
      className="no-data"
    >
      No activity found
    </div>
  </div>
  <div
    className="activity-and-progress"
  >
    <div
      className="active-times-widget"
    >
      <div
        className="active-times-header"
      >
        Active Times
      </div>
      <div
        className="active-times-chart"
      >
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#3182bd",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            12 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            2 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            4 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            6 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#3182bd",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#dfdfef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#6baed6",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            8 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            10 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#08306b",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#dfdfef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            12 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#6baed6",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            2 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#dfdfef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#6baed6",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            4 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#6baed6",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#3182bd",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#3182bd",
              }
            }
          />
          <div
            className="time-label"
          >
            6 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#08306b",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#dfdfef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#6baed6",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#9ecae1",
              }
            }
          />
          <div
            className="time-label"
          >
            8 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#3182bd",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            10 pm
          </div>
        </div>
        <div
          className="days-of-week"
        >
          <div
            className="day"
          >
            Sun
          </div>
          <div
            className="day"
          >
            Mon
          </div>
          <div
            className="day"
          >
            Tues
          </div>
          <div
            className="day"
          >
            Wed
          </div>
          <div
            className="day"
          >
            Thur
          </div>
          <div
            className="day"
          >
            Fri
          </div>
          <div
            className="day"
          >
            Sat
          </div>
        </div>
      </div>
      <div
        className="active-times-popular"
      >
        <div
          className="active-times-popular-header"
        >
          Most Popular Time
        </div>
        <div
          className="active-times-popular-content"
        >
          Monday
          , 
          12 pm
           – 
          2 pm
        </div>
      </div>
    </div>
    <div
      className="site-admin-card site-admin-labeled-value user-course-enrollment-progress"
    >
      <div
        className="label"
      >
        Course Progress
      </div>
      <div
        className="value"
      >
        <div
          className="circular-progress"
        >
          <svg
            height={200}
            viewBox="0 0 36 36"
            width={200}
          >
            <path
              d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831"
              fill="none"
              stroke="#eee"
              strokeDasharray="100, 100"
              strokeWidth="2"
            />
            <path
              d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831"
              fill="none"
              stroke="#3fb34f"
              strokeDasharray="55, 100"
              strokeWidth="2"
            />
          </svg>
          <div
            className="value"
            style={
              Object {
                "fontSize": 66.66666666666667,
                "top": 50,
                "width": 200,
              }
            }
          >
            <span
              className="number"
            >
              55
            </span>
            <span
              className="percent"
              style={
                Object {
                  "fontSize": 33.333333333333336,
                }
              }
            >
              %
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
